index.html

```diff
@@ -76,6 +76,258 @@ <h1>I’m Joseph Glad to see you!</h1>
+
+            <ul class="work__lang">
+              <li><a href="#" title="html" class="lang-1">html</a></li>
+              <li><a href="#" title="css" class="lang-2">css</a></li>
+              <li>
+                <a href="#" title="javascript" class="lang-3">javascript</a>
+              </li>
+            </ul>
+          </div>
+          <a href="#" class="btn btn-enabled">See Project</a>
+        </div>
+      </section>
+      <section class="works">
+        <div class="works__image">
+          <img src="/img/Snapshoot-Portfolio3.png" alt="work 1" />
+        </div>
+        <div class="work__description">
+          <h2>Tonic</h2>
+          <div class="short_desc">
+            <h4>canopi</h4>
+            <span></span>
+            <h4>Back End Dev</h4>
+            <span></span>
+            <p>2015</p>
+          </div>
+          <div class="long_desc">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <ul class="work__lang">
+              <li><a href="#" title="html" class="lang-1">html</a></li>
+              <li><a href="#" title="css" class="lang-2">css</a></li>
+              <li>
+                <a href="#" title="javascript" class="lang-3">javascript</a>
+              </li>
+            </ul>
+          </div>
+          <a href="#" class="btn btn-enabled">See Project</a>
+        </div>
+      </section>
+      <section class="works">
+        <div class="works__image">
+          <img src="/img/Snapshoot-Portfolio4.png" alt="work 1" />
+        </div>
+        <div class="work__description">
+          <h2>Multi-Post Stories</h2>
+          <div class="short_desc">
+            <h4>canopi</h4>
+            <span></span>
+            <h4>Back End Dev</h4>
+            <span></span>
+            <p>2015</p>
+          </div>
+          <div class="long_desc">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <ul class="work__lang">
+              <li><a href="#" title="html" class="lang-1">html</a></li>
+              <li><a href="#" title="css" class="lang-2">css</a></li>
+              <li>
+                <a href="#" title="javascript" class="lang-3">javascript</a>
+              </li>
+            </ul>
+          </div>
+          <a href="#" class="btn btn-enabled">See Project</a>
+        </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-container">
+          <div class="about-me__title">About Myself</div>
+          <div class="about-me__description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </div>
```

Comment:
- [ ] It would be better to use `h2` tag or `h3` tag for the title `About me` to improve our structure and semantics.
- [ ] The same thing for the `description`. Let's use `p` tag for it.